Umweltzone/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,348 @@
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" xmlns:ns2="http://schemas.android.com/apk/res-auto">
+
+    <!--Action Bar items-->
+    <string name="action_cities">Steden</string>
+    <string name="action_share_app">Deel deze app &#8230;</string>
+    <string name="action_changelog">Versie informatie</string>
+
+    <!-- Share intent -->
+    <string name="share_intent_message">
+        <![CDATA[Kent u de milieuzone al? Deze Android-app bevat alle informatie over milieuzones en dieselverboden in Duitsland.
+        \nDownload hier gratis: https://play.google.com/store/apps/details?id=de.avpptr.umweltzone]]>
+    </string>
+
+    <!-- Map -->
+    <string name="map_my_location_activation_hint">
+        Tik hier om uw eigen positie op de kaart te tonen
+    </string>
+
+    <!-- FeedbackActivity -->
+    <string name="feedback_activity_title">Feedback</string>
+
+    <!-- AboutActivity -->
+    <string name="appinfo_application_logo_description">
+        Logo van de &#8220;Umweltzone&#8221; App
+    </string>
+    <string name="about_activity_title">Over</string>
+    <string name="appinfo_subtitle">
+        Een Android-app voor milieuzones en dieselverboden in Duitsland
+    </string>
+
+    <!-- AboutActivity / Copyright -->
+    <string name="appinfo_copyright_title">Idee en realisatie</string>
+    <string name="appinfo_copyright_content">
+        (&#169;) Tobias Preuß, 2013 &#8211; 2019
+    </string>
+
+    <!-- AboutActivity / Description -->
+    <string name="appinfo_description_title">Beschrijving</string>
+    <string name="appinfo_description_content">
+        Deze applicatie toont de locatie van zones met lage emissie (LEZ) en Diesel
+        verbodszones (DPZ) en biedt meer informatie over dit onderwerp.
+        Rijden in een LEZ is beperkt tot voertuigen die zijn getagd met een
+        specifieke sticker. De sticker komt overeen met het emissieniveau van de
+        voertuig. Voertuigen die niet voldoen aan een bepaalde dieseluitlaatemissie
+        standaard zijn beperkt om een ​​DPZ in te voeren. Het doel van de inreisbeperkingen is
+        om de luchtkwaliteit in steden te verbeteren.
+        Fijne deeltjes moeten worden verminderd om de gezondheid van de burger te behouden. &#8212;
+        Naast de kaart en een lijst met steden is er een detailscherm dat
+        bevat individuele informatie voor de geselecteerde stad.
+        Veelgestelde vragen zijn te vinden via de
+        &#8220; FAQ &#8221; menu onderdeel. Bovendien zijn er links naar websites die
+        meer informatie bieden over LEZ, DPZ en gerelateerde onderwerpen.
+    </string>
```

Comment:
Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts.